### DISCUSSION TITLE
[预览]v1.12.18

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2274] [分享 / Share]
- macOS 优化:
  - 修复搜索栏中输入法无法选词. (#2197)
  - 默认禁用 `使用细滚动条`. (新装的才是默认禁用, 之前开着的 macOS 用户需要手动关闭) (#2207)
- 尝试修复 `播放前显示封面` 在 `bwp-video` 页面下失效. (#2103)
- 修复英文下登录文字溢出. (#2208)
- 修复已关闭评论区的样式. (#2185)
- `简化直播间`的`入场通知`也会屏蔽刚进入时的系统提示. (#2192)
- 倍速相关功能支持在番剧区使用. (#1992)
- `简化首页`改进:
  - 适配新版首页 (#2209, #2271)
  - 增加`个性化推荐`开关, 可以控制清爽首页中显示热门内容还是推荐视频
  - 显示推荐视频时, 支持"换一批" (#2168)
  - `热门`和`动态`会伸长填满剩余空间, 不过还没完全弄好, 窄屏幕下排行榜会溢出

<details>
<summary><strong>v1.12.17</strong></summary>

- 试着在番剧区恢复了一些快捷键. (#1992)

</details>

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.12.18-preview'>[预览]v1.12.18</a>.</em>

**Comment by xingsuxingchen** [comment]
为什么我用直接下载的视频竟然是.txt的格式，修改扩展名为MP4后竟然可以观看了？这是什么问题？
